exercises/book-store/canonical-data.json

```diff
@@ -0,0 +1,76 @@
+{
+   "book-store": {
+      "description": "Calculate lowest price for shopping basket only containing books from a single series",
+      "cases": [
+         {
+            "description": "TestSingleBookPrice",
+            "expected": 8.0
+         },
+         {
+            "description": "TestBasketContainsMyBooks",
+		    "mybasket": "3",
+            "expected": "3"
+         },
+         {
+            "description": "TestTotalBasketPriceSingleBook",
+            "basket": "1",
+            "expected": 8.0
+         },
+         {
+            "description": "TestTotalBasketPriceTwoofSameBook",
+            "basket": "22",
+            "expected": 16.0
+         },
+         {
+            "description": "TestTotalBasketPriceEmptyBasket",
+            "basket": "",
+            "expected": 0.0
+         },
+         {
+            "description": "TestBasketWith2DifferentBooks",
+            "basket": "12",
+            "expected": 15.2
+         },
+         {
+            "description": "TestBasketWith3DifferentBooks",
+            "basket": "123",
+            "expected": 21.60
+         },
+         {
+            "description": "TestBasketWith4DifferentBooks",
+            "basket": "1234",
+            "expected": 25.60
+         },
+         {
+            "description": "TestBasketWith5DifferentBooks",
+            "basket": "12345",
+            "expected": 30.00
+         },
+         {
+            "description": "basket 11223345",
+            "basket": "11223345",
+            "expected": 51.20
+         },
+         {
+            "description": "basket 112233445",
+            "basket": "112233445",
+            "expected": 55.60
+         },
+         {
+            "description": "basket 1122334455",
+            "basket":  "1122334455",
+            "expected": 60.00
+         },
+         {
+            "description": "basket 11223344551",
+            "basket": "11223344551",
+            "expected": 68.00
+         },
+         {
+            "description": "basket 112233445512",
+            "basket": "112233445512",
```

Comment:
previous discussion: https://github.com/exercism/x-common/pull/465#discussion_r91584069 - would be good if this were an array

> String type comes with the added benefit of TStringHelper which makes parsing the basket a little easier.

Sorry, I'm not familiar with what a TStringHelper is nor what language it's associated with. We'd like the canonical JSON files to be independent of any one language.